### PR TITLE
Minor language and url updates

### PR DESCRIPTION
**Pull request by nathan-duddles**

### CODE REVIEW

File: _labs/lab02.markdown [hunk 1]
```diff
@@ -32,8 +32,10 @@ reference for most of the LLVM API used in this lab and also throughout the cour
 
 ### Setup
 
-+ Navigate to the `lab2` folder on your computer and open it in VS code.
-+ Press F1 key to display a search bar, search and select `Reopen in Container`.
++ Open the lab2 folder in VS Code, using the terminal command `code <lab2 directory>` 
+or 'Open Folder' option in VS Code.
++ Make sure the Docker is running on your machine.
++ Open the VS Code [Command Palette](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_command-palette); search and select `Reopen in Container`.
```
Comment (hXtreme) [inline _labs/lab02.markdown]: Could you move the actual link to the bottom of the document with the other links.

File: _labs/lab02.markdown [hunk 2]
```diff
@@ -32,8 +32,10 @@ reference for most of the LLVM API used in this lab and also throughout the cour
 
 ### Setup
 
-+ Navigate to the `lab2` folder on your computer and open it in VS code.
-+ Press F1 key to display a search bar, search and select `Reopen in Container`.
++ Open the lab2 folder in VS Code, using the terminal command `code <lab2 directory>`
```
Comment (hXtreme) [inline _labs/lab02.markdown]: put lab2 inside backticks 

can you put the shell command on it's own code block, see how we show shell commands in the coursevm doc for comparison.

**Review by hXtreme — CHANGES_REQUESTED** [review]
Needs some minor edits.

File: _labs/lab02.markdown (hunk 1 above)
Comment (nathan-duddles) [reply _labs/lab02.markdown]: Good catch. Made the update.

File: _labs/lab02.markdown (hunk 2 above)
Comment (nathan-duddles) [reply _labs/lab02.markdown]: Made these changes too

**Review by hXtreme — APPROVED** [review]

**Pull request merged**